pandas/conftest.py

```diff
@@ -1347,6 +1347,22 @@ def any_signed_nullable_int_dtype(request):
     return request.param
 
 
+@pytest.fixture(params=tm.SIGNED_EA_INT_DTYPES + tm.FLOAT_EA_DTYPES)
+def any_signed_nullable_numeric_dtype(request):
+    """
+    Parameterized fixture for any signed nullable integer dtype
+    and float ea dtypes.
+
+    * 'Int8'
+    * 'Int16'
```

Comment:
i would rather have any_nullable_real_dtype (e..g. include unsigned ints), this seems an oddball.

Reply:
i don't think unsigned ints are supposed to be used here. It wouldn't make sense to use unary operators like `+` and `-` on an unsigned dtypes right?

Reply:
and that's exatly the reason to test this

Reply:
pos and abs should for sure work and give consistent results. uints *work* as well (though the results may surprise you, they are consistent with how uints work generally). so pls test these.

Reply:
if that's the case, should we separate the tests for uints? I think a single test handling all real dtypes would be too big.I would split the tests into signed ints, unsigned ints, and floats.

Reply:
i don't see why, the way you have your tests now it should just work

Reply:
I'm not sure how the tests for uints should be implemented? For test cases where there are negative numbers in the input, you can't create a series. Should I just use `pytest.raises`  in these cases?

Reply:
numpy gives a result so this will as well (it wraps around but that's irrelevant)